apps/wallet-dashboard/components/dialogs/migration/MigrationDialog.tsx

```diff
@@ -54,6 +55,11 @@ export function MigrationDialog({
                     onSuccess(tx.digest);
                     setTxDigest(tx.digest);
                     setView(MigrationDialogView.TransactionDetails);
+                    ampli.migration({
+                        basicOutputObjects: basicOutputObjects.length,
+                        nftOutputObjects: nftOutputObjects.length,
+                        isTimelocked: isTimelocked,
```

Comment:
not sure we care abut the timelock here, if ismigratable then it cant be locked at the time of migration